src/read.c

```diff
@@ -535,8 +535,9 @@ static avifBool avifCodecDecodeInputFillFromDecoderItem(avifCodecDecodeInput * d
     }
 
     const avifProperty * lselProp = avifPropertyArrayFind(&item->properties, "lsel");
-    item->progressive = (a1lxProp && !lselProp); // Progressive images offer layers via the a1lxProp, but don't specify a layer selection with lsel.
-    if (lselProp) {
+    // Progressive images offer layers via the a1lxProp, but don't specify a layer selection with lsel.
+    item->progressive = (a1lxProp && lselProp && (lselProp->u.lsel.layerID == 0xFFFF));
```

Comment:
James: If we want to treat absent and layer_id == 0xffff equivalently for backward compatibility, we just need to change this line to:
```
    item->progressive = (a1lxProp && (!lselProp || (lselProp->u.lsel.layerID == 0xFFFF)));
```

Reply:
I would lean towards compatibility. This change on this line made me wonder. I didn't read through the current spec, but was basing my comment on the linked diff.

Reply:
In talking about this over chat, we're not mandating a lsel box in this change, just potentially changing the rendering so this is ok if we wanted to deal with the possibility of bug reports or questions about why particular files aren't being rendered consistently.